docs/reference/action_list.rst

```diff
@@ -80,6 +80,14 @@ Here is an example::
                 ],
             ])
 
+            // you can use Symfony Data Transformers to transform value into the data form expected by your
+            // application. For example, in a Value Object
+            ->add('permission', 'choice', [
+                'editable' => true,
+                'choices' => Permission::choices(),
+                'data_transformer' => new PermissionDataTransformer(),
+            ])
+
```

Comment:
@OskarStark will this example be enough?

Reply:
I would move this to a dedicated section, show this + `Permission` value object + `PermissionDataTransformer`, WDYT?